views/components/MarkdownRenderer.jsx

```diff
@@ -168,18 +170,21 @@ const MarkdownRenderer = ({ content, loading, isGPT }) => {
         <Spinner />
       ) : (
         <>
-          <StyledMarkdown
-            isGPT={isGPT}
-            chatMode
-            remarkPlugins={[remarkGfm]}
-            components={{
-              code: CodeBlock,
-            }}
-          >
-            {content}
-          </StyledMarkdown>
+          {content && (
+            <StyledMarkdown
+              isGPT={isGPT}
+              chatMode
+              remarkPlugins={[remarkGfm]}
+              components={{
+                code: CodeBlock,
+              }}
+            >
+              {content}
+            </StyledMarkdown>
+          )}
+          <ErrorMessage>{error}</ErrorMessage>
```

Comment:
Do we need to make it conditional? Like `{ error && (<ErrorMessage...`?

Reply:
Sure, I had added it at the beginning, but it seems I accidentally removed it after making some changes